### ISSUE TITLE
User custom field in email template sent to admin

### ISSUE THREAD

**Issue by jgkiefer** [joomla/joomla-cms#43439]
### Steps to reproduce the issue

Unable to place a user custom field in the email template sent to admin.

### Expected result
The ability to add user custom fields in the new user email template sent to admin.

**Comment by TLWebdesign** [comment]
This would be an awesome feature for Joomla 5.2 😅

**Comment by staweb1351** [comment]
This would be an awesome feature for Joomla 5.2 😅😅<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43439">issues.joomla.org/tracker/joomla-cms/43439</a>.</sub>

**Comment by Hackwar** [comment]
You are welcome to provide a PR to implement this. If you get it ready by August 14th and have it tested by at least 2 people, it can still be added to Joomla 5.2.

**Comment by staweb1351** [comment]
Hello Hackwar
I don't understand exactly what I have to do (PR) for public relation?
Should I tell the French community about this development, on the official joomla forum? (there's already a post redirecting you here https://forum.joomla.fr/forum/joomla-5-x/questions-g%C3%A9n%C3%A9rales-ab/2062664-overdrive-formulaire-cr%C3%A9ation-de-compte-email-d-approbation-d-inscription)

thanks for your reply

**Comment by Hackwar** [comment]
PR = Pull Request. Someone has to write the code and then hand it in as a pull request here. Then 2 different people have to test it and when everything is fine, we can merge that into 5.2.

**Comment by staweb1351** [comment]
OK thanks Hackwar, but I'm not a coder. But I'm willing to test :)

**Comment by VacancesBleues** [comment]
Hi, I've just asked an acquaintance on Discord to help us with his expertise in development, I'll keep you informed.

**Comment by jgkiefer** [comment]
Thats terrific! Thanks for keeping us informed.
I would be willing to help test it out if needed.

________________________________

**Comment by VacancesBleues** [comment]
Hi community,

I'm pleased to introduce Janson, a valuable contact I met on Discord during discussions about Joomla. Janson has several years of experience in development and would like to contribute to the improvement of Joomla for the next version by working on the CMS code. His passion and skills will be a valuable asset to our project.

**Comment by upmanager** [comment]
I developed some plugins and UserPro and other plugins. No worries.

**Comment by jgkiefer** [comment]
Hi Jason thanks for helping out!
________________________________

**Comment by upmanager** [comment]
I want to work with you. Can we discuss more?

**Comment by VacancesBleues** [comment]
Hi Jason,

It's me Deedrah :)  Thanks again for your interest, of course we can discuss this further

In the Joomla backoffice, we add to the user a custom field (e.g. employee registration number)
On the frontoffice, the user creates an account
an automatic email is sent to the administrator via the email ID template  [com_users.registration.admin.verification_request]

Currently, we have native insert tags: name, sitename, email, username, activate
We'd also need custom fields (e.g. employee registration number)

We're here to listen !

**Comment by upmanager** [comment]
Yeah, I totally understand what you mean. This project is similar with UserPro plugin and with my experience, I can complete this project perfectly.
Thank you

**Comment by upmanager** [comment]
Let's great group chat on slack or discord

**Comment by VacancesBleues** [comment]
Yes! except that here, it's free and will be accessible to the entire worldwide Joomla community, too powerful 🚀
ready to listen and to test

**Comment by upmanager** [comment]
Great. I want to work asap

**Comment by Hackwar** [comment]
Just to remind you that we have the Joomla Mattermost for real time chat. Also the issues in GitHub are not exactly meant for chats. If you want to contribute, you are more than welcome to do so, but simply just start with the work, create a PR and then we discuss from that PR. Keep in mind the contribution guidelines.

**Comment by upmanager** [comment]
This is paid job or internship?

**Comment by chmst** [comment]
It is neither paid nor internship. 
Just make a PR if you want to contribute. What makes you think that this is paid job?
